addons/fleet/views/fleet_vehicle_views.xml

```diff
@@ -326,9 +326,10 @@
                 <field name="id"/>
                 <templates>
                     <div t-name="activity-box">
-                        <img t-att-src="activity_image('fleet.vehicle', 'image_128', record.id.raw_value)" role="img" t-att-title="record.id.value" t-att-alt="record.id.value"/>
-                        <div>
-                            <field name="license_plate"/> : <field name="model_id"/>
+                        <img class="rounded-circle" t-att-src="activity_image('fleet.vehicle', 'image_128', record.id.raw_value)" role="img" t-att-title="record.id.value" t-att-alt="record.id.value"/>
```

Comment:
It's not desirable to have rounded-circle on vehicle brand image.

Reply:
removed